localstripe/server.py

```diff
@@ -25,10 +25,10 @@
 from aiohttp import web
 
 from .resources import Charge, Coupon, Customer, \
-                       Event, Invoice, InvoiceItem, PaymentIntent, \
-                       PaymentMethod, Plan, Product, Refund, SetupIntent, \
-                       Source, Subscription, SubscriptionItem, TaxRate, \
-                       Token, extra_apis, store
+    Event, Invoice, InvoiceItem, PaymentIntent, \
+    PaymentMethod, Plan, Product, Refund, SetupIntent, \
+    Source, Subscription, SubscriptionItem, TaxRate, \
+    Token, extra_apis, store
```

Comment:
I think this PR doesn't add or remove imports anymore.
If it's true, it shouldn't modify these lines.

Reply:
Sorry, the linter :/